composer.json

```diff
@@ -9,17 +9,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "webmozart/console": "^1.0-beta5",
+        "php": ">=7.0",
         "webmozart/path-util": "^2.3",
         "symfony/filesystem": "^3.2",
         "symfony/process": "^3.2",
         "symfony/finder": "^3.2",
         "padraic/phar-updater": "^1.0.3",
-        "nikic/php-parser": "^3.0"
+        "nikic/php-parser": "^3.0",
+        "mnapoli/silly": "^1.6",
+        "symfony/console": "^3.2"
```

Comment:
There was a specific problem with `symfony/console` at 3.0, pegging it to 3.2 resolves that on `--prefer-lowest` builds.